src/styles/blocks/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  width: $card-dimension;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    color: $description-color-text;
+  }
```

Comment:
Why do you need this container?
1 - container is something that wrapping your element
```
  <div class="container">
    <div class="content"></div>
  </div>
 ```
 
 2 - all this styles can be applied to `card`, no need to create extra div for it

Reply:
not needed because I'm positioned content in container